### PR TITLE
Add Greek font table and GR translation

### PR DESCRIPTION
**Pull request by Devnol**

### CODE REVIEW

**Comment by Devnol** [comment]
Please do not merge yet as there are some untranslated strings for whose I do not have context yet, waiting for the build output to see them.

File: Translations/font_tables.py [hunk 1]
```diff
@@ -325,6 +325,70 @@ def get_font_map_latin_extended() -> Dict[str, bytes]:
     }
     return font
 
+#TODO: Add Greek Font
```
Comment (alvinhochun) [inline Translations/font_tables.py:328]: Why the TODO?

File: Translations/font_tables.py [hunk 2]
```diff
@@ -749,6 +814,69 @@ def get_small_font_map_latin_extended() -> Dict[str, bytes]:
     }
     return font
 
+#TODO: Add small greek font
```
Comment (alvinhochun) [inline Translations/font_tables.py:817]: Same as above

File: Translations/font_tables.py [hunk 3]
```diff
@@ -772,7 +900,7 @@ def get_small_font_map_cyrillic() -> Dict[str, bytes]:
         "А": b"\x7e\x09\x09\x09\x7e\x00",
         "Б": b"\x7f\x49\x49\x49\x31\x00",
         "В": b"\x7f\x49\x49\x49\x36\x00",
-        "Г": b"\x7f\x01\x01\x01\x01\x00",
+        "Г": b"\x7f\x49\x49\x49\x36\x00",
```
Comment (alvinhochun) [inline Translations/font_tables.py:903]: Is this change accidental?

**Comment by alvinhochun** [comment]
You should probably add `greek` to https://github.com/Ralim/IronOS/blob/bc7d106c9f19bb6a2132b196b2e3054323f0322c/Translations/TranslationEditor.html#L308 too

File: Translations/font_tables.py (hunk 1 above)
Comment (Devnol) [reply Translations/font_tables.py:328]: Oh sorry, chronic brain damage from watching Stanley Kubrick

File: Translations/font_tables.py (hunk 2 above)
Comment (Devnol) [reply Translations/font_tables.py:817]: Same as above

File: Translations/font_tables.py (hunk 3 above)
Comment (Devnol) [reply Translations/font_tables.py:903]: Yes I accidentally cmd+v'd instead of cmd+c'ing and it got changed into a B

**Comment by Devnol** [comment]
Will do right now.

**Comment by alvinhochun** [comment]
Guess this has to wait for @Ralim to approve the workflow. In the meantime you can probably enable GitHub Actions in your fork to test things out now.

**Comment by Devnol** [comment]
I was trying to test the changes locally but make somehow only builds english (see #1120)

**Comment by alvinhochun** [comment]
Enabling Actions in your fork is simple enough, just visit https://github.com/Devnol/IronOS/actions and click the button.

**Comment by Devnol** [comment]
ik I did it I just need to commit again to trigger the workflow

**Comment by Devnol** [comment]
Any idea where the `IdleSetString` and `IdleTipString` strings can be found inside the UI?

**Comment by alvinhochun** [comment]
> Any idea where the `IdleSetString` and `IdleTipString` strings can be found inside the UI?

Probably in the detailed idle screen (need to enable it in settings), or might've been obsolete after someone revamped the detailed idle screen

**Comment by Devnol** [comment]
Nope, not there, it was probably removed as I can't find a reference to it in any of the source files, so I'll just leave it as-is

**Comment by Ralim** [comment]
Oops sorry for forgetting to enable actions.
Should be good from now one 😓

**Comment by Ralim** [comment]
> Nope, not there, it was probably removed as I can't find a reference to it in any of the source files, so I'll just leave it as-is

I believe they are not used at the moment. I shall try and clean these up later on

**Comment by Ralim** [comment]
heya @Devnol not sure how you feel on this, but maybe we should name the file after the language code for [Greek](http://www.lingoes.net/en/translator/langcode.htm) with i think is `EL` weirdly?

Just thinking if we should head towards these to prevent collisions? Though not super firm on this?

**Comment by Devnol** [comment]
both EL and GR are valid, usually EL is used for the language and GR for the country because there's also the el_CY locale used in cyprus which just has different measuring units and doesn't apply to IronOS. There shouldn't be any  collision but if you wanna be precise EL is the correct code. Fyi it's not weird EL comes from the greek name for the language, which is Ellinika (same root as Hellenic). If you want, I'll fix it once I get home.

**Comment by alvinhochun** [comment]
I'd prefer `EL` or `EL_GR` if you want, just to keep it like standard BCP 47 language tags (with dash replaced by underscore). You can see some other translations already follow this convention (JA_JP, NL_BE, SR_CYRL, SR_LATN, YUE_HK, ZH_CN, ZH_TW).

**Pull request merged**